libraries/AP_Proximity/AP_Proximity_Backend.cpp

```diff
@@ -67,9 +64,9 @@ uint8_t AP_Proximity_Backend::get_object_count() const
 // returns false if no angle or distance could be returned for some reason
 bool AP_Proximity_Backend::get_object_angle_and_distance(uint8_t object_number, float& angle_deg, float &distance) const
```

Comment:
I think maybe get_object_angle_and_distance should also be moved into Boundary_3D.